inst/js/rwebppl

```diff
@@ -62,4 +62,10 @@ function main() {
 
 }
 
-main();
+try {
+  main();
+} finally {
+  var argv = parseArgs(process.argv.slice(2));
+  var finishFile = argv._[2];
```

Comment:
This would also be much improved by using Optimist (see above) -- then you could say
`var finishFile = argv.finish`